_inc/client/components/module-settings/modules-per-tab-page.jsx

```diff
@@ -55,7 +55,13 @@ export const AllModuleSettings = React.createClass( {
 			case 'monitor':
 				return ( <MonitorSettings module={ module } { ...this.props } /> );
 			case 'scan':
-				return '' === module.configure_url ? null : (
+				return '' === module.configure_url ? (
+					<div>
+						{
+							__( 'Upgrade Jetpack and our state-of-the-art security scanner will hunt out malicious files and report them immediately so that you’re never unaware of what is happening on your website.' )
```

Comment:
There's a weird quote here in `we're`. I think it should be replaced by regular quote